src/arrayMethodJoin.js

```diff
@@ -6,6 +6,20 @@
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
     // write code here
+    const sep = separator !== undefined ? String(separator) : ',';
```

Comment:
Instead of this use default value for parameter https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters